arrow-fx-test/src/main/kotlin/arrow/test/util/LawUtils.kt

```diff
@@ -7,19 +7,12 @@ import arrow.test.generators.tuple5
 import arrow.typeclasses.Eq
 import io.kotlintest.Matcher
 import io.kotlintest.Result
-import io.kotlintest.TestContext
 import io.kotlintest.properties.Gen
 import io.kotlintest.should
 
 fun throwableEq() = Eq { a: Throwable, b ->
   a::class == b::class && a.message == b.message
 }
-
-data class Law(val name: String, val test: suspend TestContext.() -> Unit)
-
-fun <A> A.equalUnderTheLaw(b: A, eq: Eq<A>): Boolean =
-  eq.run { eqv(b) }
-
 fun <A> A.shouldBeEq(b: A, eq: Eq<A>): Unit = this should matchUnderEq(eq, b)
```

Comment:
Shouldn't all this file belong in arrow-core-test?

Reply:
Yes, had this here because I wanted to see if this even compiles before moving it